jupyter-book/chromatin_accessibility/introduction.ipynb

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
<p>I'd consider shortening the title</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/single-cell-best-practices/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>What about dropping the first part? So just: Single-cell ATAC-seq and the 10x Multiome assay</p><p> </p>

Reply:
This or just renaming it to a more general "experimental assays"